cmd/kubermatic-api/main.go

```diff
@@ -74,9 +74,9 @@ import (
 	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/kubernetes/scheme"
-	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog"
 	"k8s.io/metrics/pkg/apis/metrics/v1beta1"
+	ctrlrconfig "sigs.k8s.io/controller-runtime/pkg/client/config"
```

Comment:
Can we pleae not use `ctrlr` as the abbreviation for "controller"? We used `ctrl` everywhere else.

Reply:
I have no pb with this

Reply:
But I do.

Reply:
I see :-) I meant that I have no pb to change it as you proposed. It's cosmetic.